src/APIBasedAuth.ts

```diff
@@ -10,18 +10,16 @@ const TOKEN_KEYS = [
   'refreshToken',
   'tokenType'
 ];
-const DEFAULT_STORAGE_KEYS: APIBasedAuth.StorageKeys = {
-  accessToken: `${API_AUTH_STORAGE_KEY}.accessToken`,
-  expiresIn: `${API_AUTH_STORAGE_KEY}.expiresIn`,
-  idToken: `${API_AUTH_STORAGE_KEY}.idToken`,
-  refreshToken: `${API_AUTH_STORAGE_KEY}.refreshToken`,
-  session: `${API_AUTH_STORAGE_KEY}.session`,
-  tokenType: `${API_AUTH_STORAGE_KEY}.tokenType`,
-  username: `${API_AUTH_STORAGE_KEY}.username`
-};
```

Comment:
:( I thought this was much more readable.

Also, looks like we lost the `APIBasedAuth.StorageKeys` type?

Reply:
the type is still there but I'll revert to the more readable one. this format just allowed us to expand to allow more storage if desired in the future